commands/kubernetes.go

```diff
@@ -1307,6 +1307,11 @@ func mergeKubeconfig(clusterID string, remote, local *clientcmdapi.Config, setCu
 		local.CurrentContext = remote.CurrentContext
 	}
 
+	context := Context
```

Comment:
I wonder if we should use something a but more descriptive that `context`? I think there are at least three distinct "contexts" in this file.